publish.sh

```diff
@@ -137,7 +137,7 @@ aws cloudformation validate-template --template-url $template > /dev/null || exi
 echo "Outputs"
 echo Template URL: $template
 echo CF Launch URL: https://${region}.console.aws.amazon.com/cloudformation/home?region=${region}#/stacks/create/review?templateURL=${template}\&stackName=PCA
-echo CLI Deploy: aws cloudformation deploy --template-file `pwd`/build/packaged.template --capabilities CAPABILITY_NAMED_IAM CAPABILITY_AUTO_EXPAND --stack-name PCA --parameter-overrides AdminEmail=YOUR_EMAIL
+echo CLI Deploy: aws cloudformation deploy --template-file `pwd`/build/packaged.template --capabilities CAPABILITY_NAMED_IAM CAPABILITY_AUTO_EXPAND --stack-name PCA --s3-bucket ${BUCKET} --parameter-overrides AdminEmail=YOUR_EMAIL
```

Comment:
I'm not in love with writing the template to the root of the specified bucket as a side effect of the deployment.. That might get messy. 
An alternative approach could be to change this to use [aws cloudformation create-stack](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html) instead, which take a `[--template-url <value>]` parameter which we can point to the template path `$template` that we already have. What do you think?